Catalog/Services/Comments/CommentsRepository.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Catalog.Services.Comments.Interfaces;
+using Common.Interfaces;
+using Common.Models.Comments;
+using Common.Validatiors.Comments;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Catalog.Services.Comments
+{
+    public class CommentsRepository : ICommentsRepository
+    {
+        public readonly ISqlChildDataAccessor<Comment> Comments;
+
+        public CommentsRepository(ISqlChildDataAccessor<Comment> dataAccessor)
+        {
+            Comments = dataAccessor;
+        }
+
+        public async Task<bool> AddComment(Comment comment)
+        {
+            var validator = new CommentValidator();
+            validator.ValidateAndThrow(comment);
+
+            var rowsAffected = await Comments.Add(comment);
+            var resul = rowsAffected == 1;
+            return resul;
+        }
+
+        public async Task<bool> DeleteComment(int id)
+        {
+            var validator = new CommentIdValidator();
+            validator.ValidateAndThrow(id);
+
+            var rowsAffected = await Comments.Delete(id);
+            var result = rowsAffected == 1;
+            return result;
+        }
+
+        public Task<bool> EditComment(Comment comment) => throw new NotImplementedException();
```

Comment:
- [ ] Add XML documentation comments:

Example:
```xml
        /// <summary>
        /// Asynchronously edit the specified Comment
        /// <para>This method calls Edit(int, string)</para>
        /// </summary>
        /// <param name="comment">Comment to change</param>
        /// <returns>Task with the operation result flag</returns>
```

- [ ] Instead of just throwing the Exception, redirect call to the Edit(int, string) method

```csharp
public async Task<int> Edit(Comment comment) => await Edit(comment.Id, comment.Message);
```